docs/admins/truenas.md

```diff
@@ -0,0 +1,40 @@
+#  TrueNAS
+
+## General
+----
+We have a NAS unit running TrueNAS at http://vault.synshop.net (http://10.0.40.30). Any member who requires some storage space for project related storage can request an account to be created for them.
+
+- There is also a folder called "stash" that is a public, general temporary space that is mounted as a shared storage by all shop computers to quickly and easily move files between devices.
+
+- The Vault pool has a folder called "members" under which each user with an active account on the NAS has a personal folder.
+- The stash folder is at the same level.
+- Additional folders can be created as needed.
+
+- There are two user groups: Members and Admins. Folders can be given Members, Admin, or selected individual permissions as needed. More groups can be created if required but is not advised.
+
+## Administration
+----
+To add a new user to the system, you need to log into the web interface (http://vault.synshop.net), which is only possible with the root user. (Credentials are in the administrative keepass).
+To add a user:
+1. Go to Accounts>User in the left hand menu
+2. Click "Add User"
+3. Fill in the actual members name, and their chosen username (default to their Discord/known-as identity when possible)
+4. The email address is optional, but preferred. Use the member's normal contact email for future compatibility.
+5. Set a password for the member, or let them set one themselves. Note that they can only change it by SSHing into the NAS.
+6. Leave the userID to the programatically assigned one.
+7. Uncheck "new primary group" and set their group to the existing "members" group.
+8. select /mnt/vault/members from the menu as their default home directory location. (the system will auto create a folder for them with their username)
+9. Submit!
+
+The unit is a Rackable Systems with 4x2TB sata drives, with a SAS cable to a 16 drive unit.
+- Drive 0 (far left) in the control unit is the OS drive at 2TB
```

Comment:
Can you add a photo with some labels?  A picture is worth a thousand words after all ;)

need a space to render the bullet points:
```suggestion
The unit is a Rackable Systems with 4x2TB SATA drives, with a SAS cable to a 16 drive unit.

- Drive 0 (far left) in the control unit is the OS drive at 2TB
```